### PR TITLE
Small Bird Bye Bye

### PR DESCRIPTION
**Pull request by dsmith328**
## About The Pull Request
Tired of Burning the Bird?
Tired of Apocolypse Bird summoning because 1/3 of the birds are ALWAYS OUT?
LOOK NO FURTHER!
I have installed HIGH EXPLOSIVES DEVICES into the Punishing Bird's regenerator, and set for a timer of 4 Minutes!
Whenever that pesky turd breaks out, he EXPLODES IN A VIOLENT (if not minuscule) explosion! Killing him instantly! [Disclaimer, explosion not included]

## Why It's Good For The Game
The bird is ALWAYS OUT! Let's PUT HIM BACK!

## Changelog
:cl:
add: IED into Punishing Bird
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/teth/punishing_bird.dm [hunk 1]
```diff
@@ -193,6 +193,14 @@
 	..()
 	Retaliate(user)
 
+/mob/living/simple_animal/hostile/abnormality/punishing_bird/breach_effect(mob/living/carbon/human/user)
+	..()
+	addtimer(CALLBACK(src, .proc/kill_bird()), 240 SECONDS)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/teth/punishing_bird.dm:198]:
```suggestion
	addtimer(CALLBACK(src, .proc/kill_bird), 240 SECONDS)
```

File: code/modules/mob/living/simple_animal/abnormality/teth/punishing_bird.dm [hunk 2]
```diff
@@ -193,6 +193,14 @@
...
+	return
+
+/mob/living/simple_animal/hostile/abnormality/punishing_bird/proc/kill_bird()
+	Destroy()
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/teth/punishing_bird.dm:202]: It is certainly NOT a good way to handle it.

Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/teth/punishing_bird.dm:202]:
```suggestion
	if(!target)
		QDEL_NULL(src)
```

Comment (EgorDinamit) [reply code/modules/mob/living/simple_animal/abnormality/teth/punishing_bird.dm:202]: Ideally there should be more checks, but eh.

Comment (dsmith328) [reply code/modules/mob/living/simple_animal/abnormality/teth/punishing_bird.dm]: Yeah, it's entirely a temp fix to the issue. I'd love if he could path back and sit back down but I don't even know if we have a codebase FOR abnormalities returning calmly on their own.

**Pull request merged**